pypeit/scripts/coadd_datacube.py

```diff
@@ -0,0 +1,452 @@
+#!/usr/bin/env python
+#
+# See top-level LICENSE file for Copyright information
+#
+# -*- coding: utf-8 -*-
+"""
+This script enables the user to convert spec2D FITS files
+from IFU instruments into a 3D cube with a defined WCS.
+"""
+
+import argparse
+from IPython import embed
+
+from astropy import wcs, units
+from astropy.io import fits
+from astropy.coordinates import AltAz, SkyCoord
+import scipy.optimize as opt
+from scipy.interpolate import interp1d
+import scipy.signal
+import numpy as np
+import copy
+
+from pypeit import msgs
+from pypeit.spectrographs.util import load_spectrograph
+from pypeit.core.parse import get_dnum
+from pypeit.core.flux_calib import load_extinction_data, extinction_correction
+from pypeit.core.procimg import grow_masked
+from pypeit import alignframe
+from pypeit import spec2dobj
+
+
+def parser(options=None):
+
+    parser = argparse.ArgumentParser(description='Read in a spec2D file and convert it to a datacube',
+                                     formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+
+    parser.add_argument('file', type = str, default=None, help='ascii file with list of spec2D files to combine')
+    parser.add_argument('--list', default=False, help='List the extensions only?',
+                        action='store_true')
+    parser.add_argument('--det', default=1, type=int, help="Detector")
+    parser.add_argument('-o', '--overwrite', default=False, action='store_true',
+                        help='Overwrite any existing files/directories')
+
+    return parser.parse_args() if options is None else parser.parse_args(options)
+
+
+def dar_fitfunc(radec, coord_ra, coord_dec, datfit, wave, obstime, location, pressure, temperature, rel_humidity):
+    (diff_ra, diff_dec) = radec
+    # Generate the coordinate with atmopheric conditions
+    coord_atmo = SkyCoord(coord_ra + diff_ra, coord_dec + diff_dec, unit=(units.deg, units.deg))
+    coord_altaz = coord_atmo.transform_to(AltAz(obstime=obstime, location=location, obswl=wave,
+                                         pressure=pressure, temperature=temperature,
+                                         relative_humidity=rel_humidity))
+    # Return chi-squared value
+    return np.sum((np.array([coord_altaz.alt.value, coord_altaz.az.value])-datfit)**2)
+
+
+def dar_correction(wave_arr, coord, obstime, location, pressure, temperature, rel_humidity,
+                   wave_ref=None, numgrid = 10):
+    """Apply a differental atmospheric refraction correction to the input ra/dec.
+    This implementation is based on ERFA, which is called through astropy
+
+    Parameters
+    ----------
+    wave_arr (ndarray):
```

Comment:
this is a mix of Google and numpy doc style

Reply:
I think you can make it Google simply by changing `Parameters` to `Args:`

Oh, and you'd have to change the Returns approach..